osd/PlatformComponentNonRedHat.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Platform component accessed by non-Red Hat person",
```

Comment:
slight better `"Platform component accessed by non-Red Hat user"`

Reply:
changed to "user"